gpkit/keydict.py

```diff
@@ -149,10 +149,14 @@ class KeySet(KeyDict):
     "KeySets are KeyDicts without values, serving only to filter and map keys"
     collapse_arrays = False
 
-    def add(self, *args):
-        "Adds all args to the keyset"
-        for arg in args:
-            self[arg] = None
+    def add(self, item):
+        "Adds an item to the keyset"
+        self[item] = None
+
+    def union(self, iterable):
+        "Adds an iterable to the keyset"
+        for item in iterable:
+            self.add(item)
```

Comment:
actually I think this should be called `update`. `set.union` returns the union but does not mutate self. Update causes a mutation, like the implementation here.